src/com/moclojer/internal/reflection.clj

```diff
@@ -0,0 +1,94 @@
+(ns com.moclojer.internal.reflection
+  (:require
+   [camel-snake-kebab.core :as csk]
+   [clojure.string :as str]
+   [com.moclojer.rq.adapters :as adapters]))
+
+(defn unpack-parameter
+  [parameter]
+  {:type (.. parameter getType getName)
+   :name (csk/->kebab-case (.getName parameter))})
+
+(defn unpack-method
+  [method]
+  {:name (csk/->kebab-case (.getName method))
+   :parameters (map unpack-parameter (.getParameters method))})
+
+(defn underload-methods
+  [arities methods]
```

Comment:
Document functions.

They're not simple enough to be understood without explanation.